lib/features/mailto/presentation/model/mailto_arguments.dart

```diff
@@ -1,22 +1,42 @@
 
+import 'package:core/utils/app_logger.dart';
 import 'package:jmap_dart_client/jmap/core/session/session.dart';
 import 'package:tmail_ui_user/main/routes/app_routes.dart';
 import 'package:tmail_ui_user/main/routes/router_arguments.dart';
 
 class MailtoArguments extends RouterArguments {
 
+  static const String _mailtoPrefix = 'mailto:';
+
   final Session session;
-  final String? emailAddress;
+  final String? mailtoUri;
+
+  MailtoArguments({required this.session, this.mailtoUri});
 
-  MailtoArguments({required this.session, this.emailAddress});
+  Map<String, String?> toMapRouter() => mapMailtoUri;
 
-  Map<String, String?> toMapRouter() {
-    return {
+  Map<String, String?> get mapMailtoUri {
```

Comment:
I think we need unit-test for this function

Reply:
> I think we need unit-test for this function

IMO, No need to write unit-test for it. Because here it is 2 different formats. It is not suitable for a unit-test.